### DISCUSSION TITLE
Redis Authentication

### DISCUSSION THREAD

**Discussion by dhesru** [ckan/ckan#6214] [Q&A]
Hi, 

I'm aware we could connect to redis by setting the url on ckan.redis.url.
But if we want authentication between ckan and redis, is there any option for that? 

E.g. I can set _requirepass password_ on _redis.conf_ and build the image. Anyway we could do the authentication from ckan?

**Comment by frafra** [comment]
I would suggest to just bind REDIS to 127.0.0.1 in the meanwhile.
